update-static-saddlebag-data.py

```
@@ -9,7 +9,7 @@
 
 # get the upload timers
 upload_timers = requests.post(
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)